libraries/SRV_Channel/SRV_Channel.h

```diff
@@ -548,4 +555,6 @@ class SRV_Channels {
     }
 
     static bool emergency_stop;
+
+    HAL_Semaphore calc_sem;
```

Comment:
I'd put a comment besides this semaphore variable to specify what it's for.  I think it's actualy to protect the "override_counter" variable from being modified by two threads at the same time.  I probably would have name the variable "override_counter_sem" but it's up to you.  In any case, let's add a comments, thanks.

Reply:
Changes applied. Better variable naming. I'm 100% guilty of naming half of my local variables temp_something. Push upcoming.